lib/src/logic.dart

```diff
@@ -580,6 +580,12 @@ class Logic {
   /// Less-than-or-equal-to.
   Logic lte(dynamic other) => LessThanOrEqual(this, other).out;
 
+  /// Greater-than.
+  Logic gt(dynamic other) => GreaterThan(this, other).out;
```

Comment:
Looks like this PR is missing implementations for `GreaterThan` and `GreaterThanOrEqual`?

Reply:
whoops nevermind, forgot it already exists!